lib/matplotlib/axes/_base.py

```diff
@@ -3422,6 +3422,7 @@ def get_xticklabels(self, minor=False, which=None):
         """
         return self.xaxis.get_ticklabels(minor=minor, which=which)
 
+    @cbook._make_keyword_only("3.3", "fontdict")
```

Comment:
But you didn't mark `minor`, does this propagate to the remaining parameters?

Reply:
Yes.  (The only way to pass `minor` positionally would be to also pass `fontdict` positionally.)